### PR TITLE
Tag cloud improvement

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #30795 .

### Summary of Changes
This PR keeps the last word of a tag and it's badge together. 

### Testing Instructions
Make some tags and install a module tags_popular. Choose the cloud lyout. 

### Actual result BEFORE applying this Pull Request
see #30795

### Expected result AFTER applying this Pull Request
Tags and their badges are kept together. If a tag consits of several words, the last word and the badge are kept together when lines break on narrow screens.

### Documentation Changes Required
no

### CODE REVIEW

File: modules/mod_tags_popular/tmpl/cloud.php [hunk 1]
```diff
@@ -49,11 +53,19 @@
...
-			<a class="tag-name" style="font-size: <?php echo $fontsize . 'em'; ?>" href="<?php echo Route::_(RouteHelper::getTagRoute($item->tag_id . ':' . $item->alias)); ?>">
-				<?php echo htmlspecialchars($item->title, ENT_COMPAT, 'UTF-8'); ?></a>
-			<?php if ($display_count) : ?>
-				<span class="tag-count badge badge-info"><?php echo $item->count; ?></span>
-			<?php endif; ?>
+			<a class="tag-name" href="<?php echo Route::_(RouteHelper::getTagRoute($item->tag_id . ':' . $item->alias)); ?>">
+				<?php if (!$display_count) : ?>
+					<?php echo htmlspecialchars($item->title, ENT_COMPAT, 'UTF-8'); ?>
+				<?php else : ?>
+					<?php $tmp = explode(' ', htmlspecialchars($item->title, ENT_COMPAT, 'UTF-8'));?>
```
Comment (sandewt) [inline modules/mod_tags_popular/tmpl/cloud.php:60]: Line 60 and 61, missing space, should be:

`<?php $tmp = explode(' ', htmlspecialchars($item->title, ENT_COMPAT, 'UTF-8')); ?>`
`<?php $lastWord = array_pop($tmp); ?>`

**Review by zero-24 — COMMENTED** [review]
Does that nowarp thing have to be in a dedicated css file? Cant this be moved up to the template in our case?

**Comment by sandewt** [comment]
The end result of this PR will be something like this.
The text will run through the frame. Is that really the intention?

**Comment by ChrisHoefliger** [comment]
Cannot get this to work, font sizes ar ignored, see just plain text.

**Pull request closed without merging**